src/plugins/discover/public/application/main/components/layout/types.ts

```diff
@@ -6,13 +6,9 @@
  * Side Public License, v 1.
  */
 
-import {
-  DataView,
-  IndexPatternAttributes,
-  Query,
-  SavedObject,
-  TimeRange,
-} from '../../../../../../data/common';
+import type { Query, SavedObject, TimeRange } from '../../../../../../data/common';
```

Comment:
Could the `data` plugin references be via `public` as well?

Reply:
sure!

Reply:
done